common.sh

```diff
@@ -26,14 +26,13 @@ if [ -z "${CONFIG:-}" ]; then
     fi
 fi
 source $CONFIG
-cat $CONFIG
 
 WORKING_DIR=${WORKING_DIR:-"/opt/dev-scripts"}
 NODES_FILE=${NODES_FILE:-"${WORKING_DIR}/ironic_nodes.json"}
 NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
 MASTER_NODES_FILE=${MASTER_NODES_FILE:-"ocp/master_nodes.json"}
 
-if [ -z "$PULL_SECRET" ]; then
+if [ ${#PULL_SECRET} = 0 ]; then
```

Comment:
This deserves a comment so people don't end up undoing it - "avoid '-z $PULL_SECRET' to ensure the secret is not logged"